local-cli/install/install.js

```diff
@@ -10,7 +11,17 @@ log.heading = 'rnpm-install';
 function install(args, config) {
   const name = args[0];
 
-  var res = spawnSync('npm', ['install', name, '--save'], spawnOpts);
+  const projectDir = config.getProjectRoots()[0];
```

Comment:
If `process.cwd()` is what the rest of the CLI uses I think going with that sounds good.